gourmet/keymanager.py

```diff
@@ -135,55 +136,64 @@ def get_key (self,txt, certainty=0.61):
         debug("End get_key",10)
         return k
 
-    def look_for_key (self, txt):
-        """handed a key, return a sorted ALIST of potential keys and
-        scores.
+    def look_for_key(self, txt: str) -> list((str, float)):
```

Comment:
https://docs.python.org/3/library/typing.html#typing.List
I think the typing hint for List needs to be different.
For Tuple as well https://docs.python.org/3/library/typing.html#typing.Tuple

Reply:
Thanks for pointing it out, I'll refresh my memory.